modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/commerce-catalogs/catalog.products.specifications.json

```
@@ -0,0 +1,212 @@
+[
+	{
+		"cpDefinitionExternalReferenceCode": "MP-TYPE",
```

```suggestion
		"cpDefinitionExternalReferenceCode": "MKT-TYPE",
```

Same for the others.